pkg/cvo/status.go

```diff
@@ -555,6 +563,16 @@ func convertErrorToProgressing(now time.Time, statusFailure error) (reason strin
 	case payload.UpdateEffectReport:
 		return uErr.Reason, uErr.Error(), false
 	case payload.UpdateEffectNone:
+		m := time.Duration(30)
```

Comment:
m is already a duration, and `time.Duration(30)` is a very small (nonsensical) value.
```suggestion
		m := time.Duration(30 * time.Minute)
```